packages/wallet/frontend/src/pages/auth/login.tsx

```diff
@@ -158,12 +158,14 @@ const LoginPage: NextPageWithLayout = () => {
             />
           </button>
         </Form>
-        <p className="mt-auto text-center font-extralight text-green dark:text-green-neon">
-          Not a customer?{' '}
-          <Link href="signup" className="font-medium underline">
-            Create an account
-          </Link>
-        </p>
+        {!FEATURES_ENABLED ? (
+          <p className="mt-auto text-center font-extralight text-green dark:text-green-neon">
+            Not a customer?{' '}
+            <Link href="signup" className="font-medium underline">
+              Create an account
+            </Link>
+          </p>
+        ) : null}
```

Comment:
The whole registration process should be blocked. Even if we do not show the link, users can still navigate to the page. On the other hand, the backend route for sign up should be blocked as well, since it can be called via cURL, Postman, etc...